postgis/postgis.sql.in

```diff
@@ -4705,6 +4705,34 @@ CREATE OR REPLACE FUNCTION ST_GeomFromKML(text)
 	LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE
 	_COST_MEDIUM;
 
+
+
+-----------------------------------------------------------------------
+-- MARC21/XML INPUT
+-----------------------------------------------------------------------
+
+-- Availability: 3.2.3
+
+CREATE OR REPLACE FUNCTION ST_GeomFromMARC21(marc21xml text)
+	RETURNS geometry
+	AS '$libdir/postgis-3','geom_from_marc21'
```

Comment:
Please name C functions after the SQL signature (`ST_GeomFromMark21` in this case)

Reply:
Function name changed #[1922331](https://github.com/postgis/postgis/pull/659/commits/19223317aca9c1e5e8be7e2bca706fee80e1def4)